oneview_redfish_toolkit/api/status_mapping.py

```diff
@@ -37,14 +37,65 @@
     }
 }
 
+HEALTH_STATUS_MAPPING = {
```

Comment:
Same comment above.

Reply:
Let's rename to HEALTH_STATE_MAPPING based on Health description in Redfish schema: "This represents the health state of this resource in the absence of its dependent resources."